run_dir/static/js/project_samples.js

```diff
@@ -539,7 +537,7 @@ function load_samples_table() {
               } else {
                 // TODO - Wire this up to the new QC page when it's ready
                 tbl_row += '<td class="'+column_id+'"><a target="_blank" data-toggle="tooltip" title="See this sample in the LIMS" '+
-                            'href="https://genologics.scilifelab.se/clarity/search?scope=Sample&query='+info[column_id]+'">'+
+                            'href="https://genologics.scilifelab.se:8080/clarity/search?scope=Sample&query='+info[column_id]+'">'+
```

Comment:
I didn't do these changes manually, I guess it's because this line was here when I started with bioinfo tab. And then it was merged automatically